app/models/job.rb

```diff
@@ -49,7 +49,7 @@ def initialize_attributes
   def check_active_on_destroy
     if self.is_active?
       _log.warn "Job is active, delete not allowed - guid: [#{guid}], userid: [#{self.userid}], name: [#{self.name}], target class: [#{target_class}], target id: [#{target_id}], process type: [#{type}], agent class: [#{agent_class}], agent id: [#{agent_id}], zone: [#{zone}]"
-      return false
+      throw :abort
```

Comment:
Can you add a test for this? I'm not sure I understand what problem this addresses. Thanks!